cuDNN/miopen: Use per-operator headers

peterbell10 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #75576
* #75575
* #75574
* #75573
* #75572
* #75571
* #75570
* #75569
* #71217
* __->__ #71216

Differential Revision: [D33949898](https://our.internmc.facebook.com/intern/diff/D33949898)

peterbell10 committed:
cuDNN/miopen: Use per-operator headers

[ghstack-poisoned]

peterbell10 committed:
Update on "cuDNN/miopen: Use per-operator headers"




[ghstack-poisoned]

peterbell10 committed:
Update on "cuDNN/miopen: Use per-operator headers"




[ghstack-poisoned]

peterbell10 committed:
Update on "cuDNN/miopen: Use per-operator headers"




[ghstack-poisoned]

peterbell10 committed:
Update on "cuDNN/miopen: Use per-operator headers"




[ghstack-poisoned]

peterbell10 committed:
Update on "cuDNN/miopen: Use per-operator headers"




[ghstack-poisoned]

peterbell10 committed:
Update on "cuDNN/miopen: Use per-operator headers"




[ghstack-poisoned]

peterbell10 committed:
Update on "cuDNN/miopen: Use per-operator headers"




[ghstack-poisoned]

peterbell10 committed:
Update on "cuDNN/miopen: Use per-operator headers"




[ghstack-poisoned]

peterbell10 committed:
Update on "cuDNN/miopen: Use per-operator headers"




[ghstack-poisoned]

peterbell10 committed:
Rebase and fix merge conflicts on "cuDNN/miopen: Use per-operator headers"




[ghstack-poisoned]

peterbell10 committed:
Update on "cuDNN/miopen: Use per-operator headers"




[ghstack-poisoned]

peterbell10 committed:
Rebase and fix merge conflicts on "cuDNN/miopen: Use per-operator headers"




[ghstack-poisoned]

peterbell10 committed:
Update on "cuDNN/miopen: Use per-operator headers"




[ghstack-poisoned]

malfet commented:
@malfet has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33949898).

peterbell10 committed:
Rebase on master on "cuDNN/miopen: Use per-operator headers"


Differential Revision: [D33949898](https://our.internmc.facebook.com/intern/diff/D33949898)

[ghstack-poisoned]

peterbell10 committed:
Rebase and fix merge conflicts on "cuDNN/miopen: Use per-operator headers"


Differential Revision: [D33949898](https://our.internmc.facebook.com/intern/diff/D33949898)

[ghstack-poisoned]

peterbell10 committed:
Update on "cuDNN/miopen: Use per-operator headers"


Differential Revision: [D33949898](https://our.internmc.facebook.com/intern/diff/D33949898)

[ghstack-poisoned]

peterbell10 committed:
Rebase and fix merge conflicts on "cuDNN/miopen: Use per-operator headers"


Differential Revision: [D33949898](https://our.internmc.facebook.com/intern/diff/D33949898)

[ghstack-poisoned]

peterbell10 committed:
Rebase and fix merge conflicts on "cuDNN/miopen: Use per-operator headers"


Differential Revision: [D33949898](https://our.internmc.facebook.com/intern/diff/D33949898)

[ghstack-poisoned]

peterbell10 committed:
Rebase and fix merge conflicts on "cuDNN/miopen: Use per-operator headers"


Differential Revision: [D33949898](https://our.internmc.facebook.com/intern/diff/D33949898)

[ghstack-poisoned]

peterbell10 committed:
Rebase and fix merge conflicts on "cuDNN/miopen: Use per-operator headers"


Differential Revision: [D33949898](https://our.internmc.facebook.com/intern/diff/D33949898)

[ghstack-poisoned]

malfet commented:
@malfet has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33949898).

peterbell10 committed:
Rebase and fix merge conflicts on "cuDNN/miopen: Use per-operator headers"


Differential Revision: [D33949898](https://our.internmc.facebook.com/intern/diff/D33949898)

[ghstack-poisoned]

peterbell10 committed:
Rebase on "cuDNN/miopen: Use per-operator headers"


Differential Revision: [D33949898](https://our.internmc.facebook.com/intern/diff/D33949898)

[ghstack-poisoned]

peterbell10 committed:
Rebase and fix merge conflicts on "cuDNN/miopen: Use per-operator headers"


Differential Revision: [D33949898](https://our.internmc.facebook.com/intern/diff/D33949898)

[ghstack-poisoned]

peterbell10 committed:
Rebase and fix merge conflicts on "cuDNN/miopen: Use per-operator headers"


Differential Revision: [D33949898](https://our.internmc.facebook.com/intern/diff/D33949898)

[ghstack-poisoned]

peterbell10 committed:
Rebase and fix merge conflicts on "cuDNN/miopen: Use per-operator headers"


Differential Revision: [D33949898](https://our.internmc.facebook.com/intern/diff/D33949898)

[ghstack-poisoned]

peterbell10 committed:
Update on "cuDNN/miopen: Use per-operator headers"


Differential Revision: [D33949898](https://our.internmc.facebook.com/intern/diff/D33949898)

[ghstack-poisoned]

peterbell10 committed:
Rebase and fix merge conflicts on "cuDNN/miopen: Use per-operator headers"


Differential Revision: [D33949898](https://our.internmc.facebook.com/intern/diff/D33949898)

[ghstack-poisoned]

peterbell10 committed:
Rebase and fix merge conflicts on "cuDNN/miopen: Use per-operator headers"


Differential Revision: [D33949898](https://our.internmc.facebook.com/intern/diff/D33949898)

[ghstack-poisoned]

peterbell10 committed:
Rebase and fix merge conflicts on "cuDNN/miopen: Use per-operator headers"


Differential Revision: [D33949898](https://our.internmc.facebook.com/intern/diff/D33949898)

[ghstack-poisoned]

peterbell10 committed:
Rebase and remove unneeded _meta.h ops headers on "cuDNN/miopen: Use per-operator headers"


Differential Revision: [D33949898](https://our.internmc.facebook.com/intern/diff/D33949898)

[ghstack-poisoned]

peterbell10 committed:
Rebase and fix merge conflicts on "cuDNN/miopen: Use per-operator headers"


Differential Revision: [D33949898](https://our.internmc.facebook.com/intern/diff/D33949898)

[ghstack-poisoned]

peterbell10 committed:
Update on "cuDNN/miopen: Use per-operator headers"


Differential Revision: [D33949898](https://our.internmc.facebook.com/intern/diff/D33949898)

[ghstack-poisoned]

peterbell10 committed:
Rebase and fix merge conflicts on "cuDNN/miopen: Use per-operator headers"


Differential Revision: [D33949898](https://our.internmc.facebook.com/intern/diff/D33949898)

[ghstack-poisoned]

peterbell10 committed:
Rebase on "cuDNN/miopen: Use per-operator headers"


Differential Revision: [D33949898](https://our.internmc.facebook.com/intern/diff/D33949898)

[ghstack-poisoned]

peterbell10 committed:
Update on "cuDNN/miopen: Use per-operator headers"


Differential Revision: [D33949898](https://our.internmc.facebook.com/intern/diff/D33949898)

[ghstack-poisoned]

peterbell10 committed:
Update on "cuDNN/miopen: Use per-operator headers"


Differential Revision: [D33949898](https://our.internmc.facebook.com/intern/diff/D33949898)

[ghstack-poisoned]

peterbell10 committed:
Update on "cuDNN/miopen: Use per-operator headers"


Differential Revision: [D33949898](https://our.internmc.facebook.com/intern/diff/D33949898)

[ghstack-poisoned]

malfet commented:
@malfet has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33949898).

malfet commented:
This one is clean, landing
